fuel-vm/src/consts.rs

```diff
@@ -10,56 +11,56 @@ use std::mem;
 pub const VM_REGISTER_COUNT: usize = 64;
 
 /// Contains zero (0), for convenience.
-pub const REG_ZERO: usize = 0x00;
```

Comment:
Do we still need these VM constants if they are basically just aliasing the ones from fuel-asm?

Reply:
Not really, but these are used in many places. I'll try to remove them.

Reply:
Done